src/EditorFeatures/VisualBasicTest/ConvertForToForEach/ConvertForToForEachTests.vb

```diff
@@ -0,0 +1,533 @@
+﻿Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CodeRefactorings
+Imports Microsoft.CodeAnalysis.VisualBasic.ConvertForToForEach
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.ConvertForToForEach
+    Public Class ConvertForToForEachTests
+        Inherits AbstractVisualBasicCodeActionTest
+
+        Protected Overrides Function CreateCodeRefactoringProvider(workspace As Workspace, parameters As TestParameters) As CodeRefactoringProvider
+            Return New VisualBasicConvertForToForEachCodeRefactoringProvider()
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertForToForEach)>
```

Comment:
no explicit type vs implicit type option stuff for vb?

Reply:
VB does not have that feature.